src/app/features/vendor/vendor-view/vendor-view.component.html

```diff
@@ -0,0 +1,77 @@
+<p-confirmDialog
+  header="Confirmation"
+  icon="pi pi-exclamation-triangle"
+></p-confirmDialog>
+<div class="container">
+  <div class="nav"></div>
+  <div class="header">
+    <h1>
+      {{ vendor?.name }}
```

Comment:
Why are we using optional check here? Is this required?

Reply:
Corrected that